src/externs.ts

```diff
@@ -174,6 +174,18 @@ export function generateExterns(
     rootNamespace = moduleNameAsIdentifier(host, sourceFile.fileName);
   }
 
+  // moduleNamespace holds the namespace for this ES module, whereas rootNamespace is where we
+  // generate symbols (see below).
+  const moduleNamespace = rootNamespace;
+  // There can only be one export =, and if there is one, there cannot be any other exports.
```

Comment:
I understand your comments yet still don't see why do we need the '_', i.e. I am missing the big picture.

There must be some sort of constraint that passes in TS and is problematic in closure. Maybe we need a quick VC on Monday morning?

Reply:
SGTM. Monday is labor day though, let's aim for Tuesday :-)

Reply:
I've still reworded the comment here, hopefully it's a bit more clear.

Reply:
I understand this now after our VC, thanks! The missing piece is that we need to separate the module-locals and module-exports. With `exports = ` those it's not clear how those two are even related.